_includes/0.4/left_sidebar.html

```diff
@@ -34,7 +34,7 @@
     <div class="left-sidebar-group">
         <div class="left-sidebar-header">Example Applications</div>
         <a href="/docs/0.4/examples/gameroom/">Gameroom</a>
-        <a href="/docs/0.4/examples/grid.html">Grid</a>
+        <a href="/examples/#hyperledger-grid">Grid on Splinter</a>
```

Comment:
This looks like it gets changed, then reverted in another commit.

Reply:
Corrected (deleted from this commit; made the right change in the other commit).